src/lib/zip3.js

```diff
@@ -0,0 +1,17 @@
+/**
+* Copyright 2012-2018, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = function zip3(x, y, z) {
+    var result = new Array(x.length);
+    for(var i = 0; i < x.length; i++) {
+        result[i] = [x[i], y[i], z[i]];
+    }
+    return result;
+};
```

Comment:
Nice to pull this into `lib` but are we moving away from the model of including all of these in `lib/index` and using them from there?

Reply:
I like to keep subplot-specific lib function out of Lib (zip3 only makes sense for gl3d traces), but maybe it would less confusing to make `src/lib/index.js` collect everything in `src/lib/`. I don't have a strong opinion here.

Reply:
It's true, it's nice not to be carrying code in every build if only one subplot type uses it... but then, if it's really only `gl3d`, does it belong in `plots/gl3d/zip3.js`? You nailed my concern anyway, that I don't want to have to wonder every time I use a lib function whether I need to pull it in directly or if I can just require `lib/index` and pull all the pieces I need out from there.

Reply:
done in https://github.com/plotly/plotly.js/pull/2658/commits/5c2ddfc916d5895f416ebd0a98816fc87e244476